manipulators/gripper_interface/scripts/auto_gripper_interface_node.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python3
+
+import rospy
+from std_msgs.msg import Bool
+from time import sleep
+from datetime import datetime
+import Jetson.GPIO as GPIO

```

Comment:
It makes more sense for the pneumatic system to interface with the flight computer (RPi). You could talk to elektro to see what can be done about that

Reply:
I don't know why there is gripper interface code in this branch... @Lasgut can this code be deleted? Is the gripper interface not in it's own branch right now?

Reply:
It's apparently @alekskl01 that made this code. Maybe you can look into this @alekskl01 ?